src/Execution/ExecutionStrategy.php

```diff
@@ -79,7 +80,8 @@ public function __construct(
         $this->context        = $context;
         $this->operation      = $operation;
         $this->rootValue      = $rootValue;
-        $this->valuesResolver = new ValuesResolver();
+        // TODO: Inject the ValuesResolver instance by using a builder for this class.
+        $this->valuesResolver = new ValuesResolver(new ValueNodeCoercer());
```

Comment:
@crisu83 actually I think we can also inject $valueResolver from the container :D

Reply:
That was the point.

Reply:
That was the point.